lib/spack/external/vendor.txt

```diff
@@ -8,3 +8,4 @@ six==1.16.0
 macholib==1.16.2
   altgraph==0.17.3
 ruamel.yaml==0.17.21
+typing_extensions==4.1.1
```

Comment:
Any specific reason why we are using 4.1.1 and not 4.2.0 (just curious)?

Reply:
4.1.1 is the last version to support 3.6, 4.2.0 says it requires >= 3.7

Reply:
Oh, you're right, sorry for the noise:

https://github.com/python/typing/blob/9a39406fa45fbfa76eb5d35afb7863f5f727ad87/typing_extensions/pyproject.toml#L12

I was confused by this:

https://github.com/python/typing/blob/9a39406fa45fbfa76eb5d35afb7863f5f727ad87/typing_extensions/pyproject.toml#L38

but apparently it's just a leftover